specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2022-07-02/disk.json

```diff
@@ -720,6 +720,16 @@
           "type": "string",
           "description": "ARM id of the DiskAccess resource for using private endpoints on disks."
         },
+        "burstingEnabledTime": {
+          "type": "string",
+          "readOnly": true,
+          "format": "date-time",
+          "description": "Latest time when bursting was last enabled on a disk."
+        },
+        "readOnly": {
```

Comment:
Let us not add the readonly property in the swagger

Reply:
Removed It